### ISSUE TITLE
How can I print a shortcode of a CPF in the link field of a Price Table of Elemento? The href rended empty

### ISSUE THREAD

**Issue by josecristobal** [pojome/elementor#1616]

**Comment by arielk** [comment]
Can you please give us more details?

What exactly is CPF?

**Comment by josecristobal** [comment]
Custom Post Field. I work with Pods, and I can use it in the fields of Price Table. For example I can use in the field price the shortcode [pods field='price'] and display well. But in the field link of the Price Table, if I use a pod for example [pods field='url'] that contains for example the value 'http://ww.google.es', the table display an empty value for that field, and in the code of the web display an empty href. Please, excuse my bad english. I hope you understand me.

**Comment by arielk** [comment]
For now, the URL control does not support shortcode as well. I think we will add support for it in the future.

Thanks for your feedback!

**Comment by josecristobal** [comment]
Thanks for your help

**Comment by z3fei** [comment]
Yes, would like to see this option available too. I'm using a purchase shortcode with Easy Digital Downloads plugin.

**Comment by z3fei** [comment]
Guys, I first trying out by putting the purchase short code in the Button text of the pricing table, the Buy button comes but along with three other empty buttons. Then I put the purchase short code in the features and it works!

**Comment by cavalierlife** [comment]
This is important for me as well.

A shortcode option for the button, instead of just a url, would give us greater flexibility for using things such as Paypal recurring payments, or other complex situations for the Buy Now button. A simple url just doesn't always do the trick for payment buttons.

**Comment by smile00000** [comment]
+1 Need call to action shortcode in url link in elementor price list for my paypal buttons etc. That way we can send the customer to paypal but use our own elementor button instead of their unattractive one.

**Comment by mariobarretta** [comment]
Is there any news about how to use a shortcode how url ? 
I have the pop up shortcode and i'd like to can insert it on a text or in a url fo widget that not give us the possibility to choose the dynamic filed popup.
